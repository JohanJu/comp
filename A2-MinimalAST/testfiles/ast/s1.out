Program
  List
    Func ID="a"
      null
    Func ID="b"
      null
