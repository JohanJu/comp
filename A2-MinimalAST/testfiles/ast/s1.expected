Program
  List
    Func ID="a"
      List
    Func ID="b"
      List
